Contuity/Views/Explore/ExplorePresenter.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ExplorePresenter.swift
+//  Contuity
+//
+//  Created by Anthony on 10/18/18.
+//  Copyright © 2018 Generate. All rights reserved.
+//
+
+import Foundation
+
+protocol ExplorePresenterProtocol: PresenterProtocol {
+    /// The jots that need to be displayed
```

Comment:
What's the point of having this var AND the function?

You can just define this var and return the value we want like so:
`var jots: [Jot] {
    return try? Jot.getJots(queue: false) ?? []
}`

or something similar...